modules/core/include/opencv2/core/hal/intrin_neon.hpp

```diff
@@ -1045,9 +1045,24 @@ OPENCV_HAL_IMPL_NEON_BIN_FUNC(v_float64x2, v_absdiff, vabdq_f64)
 /** Saturating absolute difference **/
 inline v_int8x16 v_absdiffs(const v_int8x16& a, const v_int8x16& b)
 { return v_int8x16(vqabsq_s8(vqsubq_s8(a.val, b.val))); }
-inline v_int16x8 v_absdiffs(const v_int16x8& a, const v_int16x8& b)
+static inline v_int16x8 v_absdiffs(const v_int16x8& a, const v_int16x8& b)
```

Comment:
Why only this particular function made `static` ?

Reply:
Becase I want to make compiler to do forced inline this function to speed-up it's work.

Reply:
ok - but why only this _single_  one ? Why not other versions ? (such as one above for example)

Reply:
And parallel question - is there any proofs that this qualifier is somehow affects performance ? 
In other words are you sure that simple `inline` is not enough? (And two version - one with `static` and one without _actually_ produce two different results in terms of either performance or asm code)

Reply:
There is `CV_ALWAYS_INLINE` macro defined in cvdef.h however I agree that at least most of the time intrinsics are inlined without special treatment

Reply:
My point here is about consistency. I.e.  why only a _single_ function marked as `static` while rest (even other overloads of the same intrinsic)  are not. IMHO it's either all or none approach should be taken here. (Or at least meaningful comment/FIXME/TODO should be written here)

Reply:
@anton-potapov  please note: As you can see, this snippet is outdated. Please update page and see new version.

Reply:
Ok. Changed.